packages/flutter/test/material/divider_theme_test.dart

```diff
@@ -0,0 +1,234 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  test('DividerThemeData copyWith, ==, hashCode basics', () {
+    expect(const DividerThemeData(), const DividerThemeData().copyWith());
+    expect(const DividerThemeData().hashCode, const DividerThemeData().copyWith().hashCode);
+  });
+
+  test('DividerThemeData null fields by default', () {
+    const DividerThemeData dividerTheme = DividerThemeData();
+    expect(dividerTheme.color, null);
+    expect(dividerTheme.space, null);
+    expect(dividerTheme.thickness, null);
+    expect(dividerTheme.indent, null);
+    expect(dividerTheme.endIndent, null);
+  });
+
+  testWidgets('Default DividerThemeData debugFillProperties', (WidgetTester tester) async {
+    final DiagnosticPropertiesBuilder builder = DiagnosticPropertiesBuilder();
+    const DividerThemeData().debugFillProperties(builder);
+
+    final List<String> description = builder.properties
+        .where((DiagnosticsNode node) => !node.isFiltered(DiagnosticLevel.info))
+        .map((DiagnosticsNode node) => node.toString())
+        .toList();
+
+    expect(description, <String>[]);
+  });
+
+  testWidgets('DividerThemeData implements debugFillProperties', (WidgetTester tester) async {
+    final DiagnosticPropertiesBuilder builder = DiagnosticPropertiesBuilder();
+    const DividerThemeData(
+      color: Color(0xFFFFFFFF),
+      space: 5.0,
+      thickness: 4.0,
+      indent: 3.0,
+      endIndent: 2.0,
+    ).debugFillProperties(builder);
+
+    final List<String> description = builder.properties
+        .where((DiagnosticsNode node) => !node.isFiltered(DiagnosticLevel.info))
+        .map((DiagnosticsNode node) => node.toString())
+        .toList();
+
+    expect(description, <String>[
+      'color: Color(0xffffffff)',
+      'space: 5.0',
+      'thickness: 4.0',
+      'indent: 3.0',
+      'endIndent: 2.0',
+    ]);
+  });
+
+  group('Horizontal Divider', () {
+    testWidgets('Passing no DividerThemeData returns defaults', (WidgetTester tester) async {
+      await tester.pumpWidget(MaterialApp(
+        home: Scaffold(
+          body: const Divider(),
+        ),
+      ));
+
+      final RenderBox box = tester.firstRenderObject(find.byType(Divider));
+      expect(box.size.height, 16.0);
+
+      final Container container = tester.widget(find.byType(Container));
+      final BoxDecoration decoration = container.decoration;
+      expect(decoration.border.bottom.width, 0.0);
+      expect(decoration.border.bottom.color, const Color(0x1F000000));
```

Comment:
Where is Color(0x1F000000) coming from. Would it be better to test against the value it is stored as?

Reply:
The value comes from the default value that ThemeData creates. Even though I don't specify a theme, a fallback theme is created. 

I've updated the test to check against that value instead.